source/common/tls/default_tls_certificate_selector.h

```diff
@@ -8,11 +8,15 @@
 #include "source/common/tls/server_context_impl.h"
 #include "source/common/tls/stats.h"
 
+#include "absl/container/inlined_vector.h"
+
 namespace Envoy {
 namespace Extensions {
 namespace TransportSockets {
 namespace Tls {
 
+using CurveNIDSupportedVector = absl::InlinedVector<int, 3>;
```

Comment:
delete; get it from another header. This type definition should only be in one place.